packages/playground/src/App.vue

```diff
@@ -35,6 +35,13 @@ onMounted(() => {
       editorDocument.value = model.devModeGetDocument();
     },
   });
+  
+  core.use(EditorjsUI);
```

Comment:
So now UI will render before the core is initilised, should we change that so render happens on initialisation?

Reply:
done